docs_src/microservices/support/scheduler/Ch-Scheduler.md

```diff
@@ -77,7 +77,7 @@ This diagram is drawn by [diagram.net](https://app.diagrams.net/), and the sourc
     |HttpMethod|Indicates which Http verb should be used for the REST endpoint.(Only using when type is REST|
     |Path|The HTTP path at the targeted host for fulfillment of the action.(Only using when type is REST)|
 
-See [Interval](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-scheduler/2.1.0#/Interval) and [IntervalAction](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-scheduler/2.1.0#/IntervalAction) for more information.
+See [Interval](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-scheduler/{{version}}#/Interval) and [IntervalAction](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-scheduler/{{version}}#/IntervalAction) for more information.
```

Comment:
Change this to a relative link to `api/support/Ch-APISupportScheduler` and add `see Interval endpoint` since we can not link directly to it..